demos/storybook/stories/dropdown-toolbar/with-full-config.tsx

```diff
@@ -0,0 +1,88 @@
+import { DropdownToolbar, Spacer } from '@pxblue/react-components';
+import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types';
+import React from 'react';
+import { WITH_FULL_CONFIG_STORY_NAME } from '../../src/constants';
+import { text, select, number } from '@storybook/addon-knobs';
+import { AppBar, Theme, makeStyles, createStyles, useTheme } from '@material-ui/core';
+import { Menu, ArrowBack, Home, Work, Settings } from '@material-ui/icons';
+import { action } from '@storybook/addon-actions';
+import clsx from 'clsx';
+// @ts-ignore @TODO remove this once type definitions are completed for storybook-rtl-addon
```

Comment:
They should be added now, but you might need to ask @emclaug2 to publish a new version with the update